framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -525,10 +526,13 @@ private void applyDefaultsElement(final Element annotationScope, final Annotated
                 // but it doesn't work correctly and tests fail.
                 // (That whole @FromStubFile and @FromByteCode annotation
                 // logic should be replaced by something sensible.)
-            if ((ElementUtils.isElementFromByteCode(annotationScope) &&
+        	SourceChecker checker = atypeFactory.getContext().getChecker();
```

Comment:
Please don't use tabs in source code files.  It makes the diffs harder to read and the indentation inconsistent in the final file (depending on one's editor configuration).  You can configure your editor to use spaces instead of tabs.